packages/squiggle-lang/src/rescript/ReducerInterface/ReducerInterface_ExpressionValue.res

```diff
@@ -0,0 +1,60 @@
+/*
+  Irreducible values. Reducer does not know about those. Only used for external calls
+  This is a configuration to to make external calls of those types
+*/
+module AE = Reducer_Extra_Array
+module ErrorValue = Reducer_ErrorValue
+
+type rec expressionValue =
+  | EvBool(bool)
+  | EvNumber(float)
+  | EvString(string)
+  | EvSymbol(string)
+  | EvArray(array<expressionValue>)
+  | EvRecord(Js.Dict.t<expressionValue>)
+
+type functionCall = (string, array<expressionValue>)
+
+let rec show = aValue =>
+  switch aValue {
+  | EvBool(aBool) => Js.String.make(aBool)
+  | EvNumber(aNumber) => Js.String.make(aNumber)
+  | EvString(aString) => `'${aString}'`
+  | EvSymbol(aString) => `:${aString}`
+  | EvArray(anArray) => {
+      let args =
+        anArray->Belt.Array.map(each => show(each))->AE.interperse(", ")->Js.String.concatMany("")
```

Comment:
Can of worms there... 
E module itself will be going through a modification. It's better not to create an extra dependency for now. Otherwise we will be stepping on each others shoe. I am leaving the whole why not E module topic as a whole topic for a few revisions later. And I need to discuss a few things with Quinn also